### ISSUE TITLE
No provider for NgRedux

### ISSUE THREAD

**Issue by Sambego** [angular-redux/ng2-redux#268]
When using a the latest version of ng2-redux (`4.0.0-beta.7`) I get the following error:
```
"No provider for NgRedux!"
```

My app.module file looks like this: 
```TypeScript
...

interface IAppState { /* ... */ };
export const store: Store<IAppState> = createStore(
  rootReducer,
  compose(applyMiddleware(reduxLogger)));

@NgModule({
  declarations: [
    ...
  ],
  imports: [
    NgReduxModule.forRoot(),
    IonicModule.forRoot(MyApp),
  ],
  bootstrap: [IonicApp],
  entryComponents: [
    ...
  ],
  providers: [{provide: ErrorHandler, useClass: IonicErrorHandler}]
})
export class AppModule {
  constructor(ngRedux: NgRedux<IAppState>) {
    ngRedux.provideStore(store);
  }
}

```

I just generated a new ionic 2 project and added ng2 redux.
I'm using the following versions
- ng2-redux: `4.0.0-beta.7`
- redux: `3.6.0`
- angular: `2.1.1` (I also tried updating angular to the latest version, but that doesn't fix this :))

**Comment by nzjoel1234** [comment]
Hey. It looks like you have not added NgRedux to your providers array.

**Comment by Sambego** [comment]
Thanks for pointing that out @nzjoel1234 but sadly adding ngRedux to as a provider does not solve this issue. I've tried both with and without. Also the docs don't mention to add it :)

Maybe it's worth mentioning that I ran into this error on a bigger project.
I just set up the dummy project above to check if the error was related to my project. Since the error is also occurring in the dummy project I assume it has something to do with NgRedux.
